advanced/is-pattern-2/templates/analytics-dashboard/identity-server-analytics-dashboard-statefulset.yaml

```diff
@@ -0,0 +1,193 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@darsham3 for the Dashboard the ideal option is to use a Kubernetes Deployment definition. Can we use a Deployment definition in this case?

Reply:
updated